src/Sylius/Behat/Context/Ui/Admin/ManagingCustomerGroupsContext.php

```diff
@@ -179,6 +179,22 @@ public function iShouldBeNotifiedThatCustomerGroupWithThisCodeAlreadyExists()
         Assert::same($this->createPage->getValidationMessage('code'), 'Customer group code has to be unique.');
     }
 
+    /**
+     * @Then I should see message that information are incomplete in create section
+     */
+    public function iShouldSeeMessageThatInformationAreIncompleteInCreateSection(): void
+    {
+        Assert::same($this->createPage->getMessageInvalidForm(), 'Error This form contains errors');
+    }
+
+    /**
+     * @Then I should see message that information are incomplete in edit section
+     */
+    public function iShouldSeeMessageThatInformationAreIncompleteInEditSection(): void
```

Comment:
I would be for merging these two methods and use `CurrentPageResolverInterface` to resolve what page it is